metrics/api/v1/types/types.go

```diff
@@ -44,38 +45,10 @@ type Point struct {
 // TimeseriesSchema represents all the metrics and labels.
 type TimeseriesSchema struct {
 	// All the metrics handled by heapster.
-	Metrics []MetricDescriptor `json:"metrics,omitempty"`
+	Metrics []core.MetricDescriptor `json:"metrics,omitempty"`
```

Comment:
I'm a little worried that moving the definitions of the API types into the core makes it more likely that they'll be changed in the future without considering that they're part of what needs to be a stable API.

Is there any particular motivation for this change that justifies the risk?